ci/run_clang_tidy.sh

```diff
@@ -16,6 +16,9 @@ trap cleanup EXIT
 # Writing them into bazelrc
 echo "build ${BAZEL_BUILD_OPTIONS}" >> .bazelrc
 
+# TODO(lizan): remove this once https://github.com/grailbio/bazel-compilation-database/issues/25 is fixed.
+echo "build --incompatible_disable_legacy_cc_provider=false" >> .bazelrc
```

Comment:
FWIW, I only just read how this `.bazelrc` thing works today; it's pretty funky but I would much rather we switched to another build directory and created an ephemeral .bazelrc, e.g. what happens if you interrupt the local clang-tidy?

Reply:
Well the `trap EXIT` worked, though we don't need this any more as upstream issue is resolved, and we don't have ci WORKSPACE anymore.

Reply:
It worked providing you get the signal; if you nuke the Docker image or something, you don't get that.